core/registry.js

```diff
@@ -120,13 +132,12 @@ Blockly.registry.Type.BLOCK_DRAGGER =
  * it's type.
  * @template T
  */
-Blockly.registry.register = function(
-    type, name, registryItem, opt_allowOverrides) {
-  if ((!(type instanceof Blockly.registry.Type) && typeof type != 'string') ||
+const register = function(type, name, registryItem, opt_allowOverrides) {
+  if ((!(type instanceof Type) && typeof type != 'string') ||
       String(type).trim() == '') {
     throw Error(
         'Invalid type "' + type + '". The type must be a' +
-        ' non-empty string or a Blockly.registry.Type.');
+        ' non-empty string or a Type.');
```

Comment:
I think this error message should retain the namespace for legibility.

Reply:
Good call, updated.